pkg/kv/kvserver/replica_raft.go

```diff
@@ -172,7 +183,7 @@ func (r *Replica) evalAndPropose(
 			// Disallow async consensus for commands with EndTxnIntents because
 			// any !Always EndTxnIntent can't be cleaned up until after the
 			// command succeeds.
-			return nil, nil, "", roachpb.NewErrorf("cannot perform consensus asynchronously for "+
+			return nil, nil, "", writeBytes, roachpb.NewErrorf("cannot perform consensus asynchronously for "+
```

Comment:
I assume returning `writeBytes` in this error path was unintentional, since it's just getting discarded at the caller. I'll remove it in some later PR.

Reply:
Yes, I believe it was unintentional.